cornice_swagger/swagger.py

```diff
@@ -104,11 +111,26 @@ def _build_paths(self, ignore=['head', 'options'], default_tags=None,
             path = self._extract_path_from_service(service, **kwargs)
 
             for method_name, view, args in service.definitions:
-                if method_name.lower() in ignore:
+
+                if method_name.lower() in ignore_methods:
                     continue
 
                 op = self._extract_operation_from_view(view, args, **kwargs)
 
+                if ignore_ctypes and 'consumes' in op:
+                    if set(op.get('consumes')).issubset(set(ignore_ctypes)):
```

Comment:
This is useful because them we can preserve  generic views while ignoring more specific ones. 
E.g: One view supports ['application/json', 'application/json-patch+json'] while other is only ['application/json-patch+json'] and ignore list is ['application/json-patch+json']